Tags/Tag.js

```diff
@@ -4,15 +4,27 @@ import { View, Text, TextInput, TouchableOpacity } from "react-native";
 
 import styles from "./styles";
 
-const Tag = ({ label, onPress, tagContainerStyle, tagTextStyle }) => (
-  <TouchableOpacity style={[styles.tag, tagContainerStyle]} onPress={onPress}>
-    <Text style={[styles.tagLabel, tagTextStyle]}>{label}</Text>
-  </TouchableOpacity>
-);
+const Tag = ({ label, onPress, tagContainerStyle, tagTextStyle, readonly }) => {
+  const tagText = <Text style={[styles.tagLabel, tagTextStyle]}>{label}</Text>;
+
+  if (readonly) {
+    return (
+      <View style={[styles.tag, tagContainerStyle]} onPress={onPress}>
+        {tagText}
+      </View>
+    )
+  } else {
+    return (
+      <TouchableOpacity style={[styles.tag, tagContainerStyle]} onPress={onPress}>
+        {tagText}
+      </TouchableOpacity>
+    )
+  }
+}
 
 Tag.propTypes = {
   label: PropTypes.string.isRequired,
```

Comment:
`readonly` should be specified in the proptypes

Reply:
hey @dajaffe, just checking in to see if you saw this? I'm not sure why my CI didn't pick up this as a linting error.